Gemfile

```diff
@@ -38,8 +38,8 @@ gem 'momentjs-rails', '>= 2.9.0' # needed for human-friendly textual dates
 gem 'font-awesome-rails'
 
 # needed to run graders in the background
-gem 'delayed_job', github: 'dsander/delayed_job', branch: 'rails51' # FIXME: github
```

Comment:
AFAIK we use background jobs for two things: 1. graders 2. sending email.

Sending email goes through ActiveJob, where backburner is a drop in replacement for delayed_job. Conceptually we could stick with delayed_job here, but that seems unnecessarily redundant.

For graders, we shouldn't have two different methods of delaying jobs.